modules/serverless-functions-secrets-configmaps-interactively.adoc

```diff
@@ -0,0 +1,64 @@
+// Module included in the following assemblies:
+//
+// * serverless/functions/serverless-functions-accessing-secrets-configmaps.adoc
+
+[id="serverless-functions-secrets-configmaps-interactively_{context}"]
+= Modifying function access to secrets and config maps interactively
+
+You can manage the secrets and config maps accessed by your function by using the `kn func config` interactive utility.
+
+.Procedure
+
+. Run the following command in the function project directory:
++
+[source,terminal]
+----
+$ kn func config
+----
++
+Alternatively, you can specify the function project directory using the `--path` or `-p` option.
+
+. Use the interactive interface to perform the necessary operation. For example, using the utility to list configured volumes would produce an output similar to this:
++
+[source,terminal]
+----
+$ kn func config
+? What do you want to configure? Volumes
+? What operation do you want to perform? List
+Configured Volumes mounts:
+- Secret "mysecret" mounted at path: "/workspace/secret"
+- Secret "mysecret2" mounted at path: "/workspace/secret2"
+----
++
+This scheme shows all operations available in the interactive utility and how to navigate to them:
++
+[source]
+----
+kn func config
+   ├─> Environment variables
+   │               ├─> Add
+   │               │    ├─> ConfigMap: Add all key-value pairs from a config map
+   │               │    ├─> ConfigMap: Add value from a key in a config map
+   │               │    ├─> Secret: Add all key-value pairs from a secret
+   │               │    └─> Secret: Add value from a key in a secret
+   │               ├─> List: List all configured environment variables
+   │               └─> Remove: Remove a configured environment variable
+   └─> Volumes
+           ├─> Add
+           │    ├─> ConfigMap: Mount a config map as a volume
+           │    └─> Secret: Mount a secret as a volume
+           ├─> List: List all configured volumes
+           └─> Remove: Remove a configured volume
+----
+
+. Optional. Deploy the function to make the changes take effect:
++
+[source,terminal]
+----
+$ kn func deploy -p test
+----
++
+[NOTE]
```

Comment:
Here is this note specified as [NOTE], while in modules/serverless-functions-secrets-configmaps-manually.adoc the same note is specified as [IMPORTANT]

Reply:
Misplaced IMPORTANT, fixed now.